### DISCUSSION TITLE
A Wiki for the addons

### DISCUSSION THREAD

**Discussion by elmogosogo09usa** [ScratchAddons/ScratchAddons#5091] [Show and tell]
Hi I am elmoFX This is the wiki this Wiki Will be at https://scratchaddons.miraheze.org/wiki/Main_Page I Hope the see you there - ElmoFX

Content is available under [Creative Commons Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)](https://creativecommons.org/licenses/by-sa/4.0/) unless otherwise noted.

**Comment by Samq64** [comment]
There should probably be official documentation for at least some addons in the future. I think there was an issue for that somewhere.

**Reply by WorldLanguages** [reply]
Yeah, we won't be using a third party wiki system. Our docs are at scratchaddons.com/docs

**Comment by apple502j** [comment]
Please do not make a wiki using our brands without permission.

**Comment by Hans5958** [comment]
I have this idea since a while, but I haven't written it on an issue. I have now created ScratchAddons/website-v2#260 for this.

For collaborative efforts, this should also work as people can submit PRs on the docs (and you are welcome to, right now, for other parts of the website).

**Reply by WorldLanguages** [reply]
I guess this would be user-facing docs, right? Not developer facing docs.

**Reply by WorldLanguages** [reply]
Nevermind, clearly this is user-facing. And developer-facing comments for each addon would belong somewhere else, I assume.

**Reply by Hans5958** [reply]
Yeah this is for the user, just like the FAQ, although the information written can be descriptive enough to help developers alike (like a README, or it's own docs)
